src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectOptions.cs

```diff
@@ -55,6 +56,24 @@ public OpenIdConnectOptions(string authenticationScheme)
             Events = new OpenIdConnectEvents();
             Scope.Add("openid");
             Scope.Add("profile");
+
+            ClaimMaps.DeleteClaim("nonce");
```

Comment:
We try to avoid params in public APIs. We'll see how often this gets used externally.